extension/src/popup/components/WarningMessages/index.tsx

```diff
@@ -746,70 +749,78 @@ export const UnverifiedTokenWarning = ({
+            <div className="TokenWarning__flags__info">{t("Asset Info")}</div>
 
-          <Notification
-            title={t(
-              "Before you add this asset, please double-check its information and characteristics. This can help you identify fraudulent assets.",
+            {isVerifiedToken ? null : (
+              <div className="TokenWarning__flag">
+                <div className="TokenWarning__flag__icon">
+                  <img src={IconUnverified} alt="unverified icon" />
+                </div>
+                <div className="TokenWarning_flag__content">
+                  <div className="TokenWarning__flag__header TokenWarning__flag__icon--unverified">
+                    {t("Unverified asset")}
+                  </div>
+                  <div className="TokenWarning__flag__content">
+                    {t("Proceed with caution")}
+                  </div>
+                </div>
+              </div>
             )}
-            variant="warning"
-          />
-          <div className="UnverifiedTokenWarning__flags">
-            <div className="UnverifiedTokenWarning__flags__info">
-              {t("Asset Info")}
-            </div>
-            <div className="UnverifiedTokenWarning__flag">
-              <div className="UnverifiedTokenWarning__flag__icon">
-                <Icon.Info />
+            <div className="TokenWarning__flag">
+              <div className="TokenWarning__flag__icon">
+                <img src={IconNewAsset} alt="new asset icon" />
               </div>
-              <div className="UnverifiedTokenWarning__flag__content">
-                <div className="UnverifiedTokenWarning__flag__header UnverifiedTokenWarning__flags__icon--unverified">
-                  {t(
-                    "The asset is not part of Stellar Expert's top 50 assets list",
-                  )}
+              <div className="TokenWarning_flag__content">
+                <div className="TokenWarning__flag__header TokenWarning__flag__icon">
+                  {t("New asset")}
                 </div>
-                <div className="UnverifiedTokenWarning__flag__description">
-                  {t("This asset is not part of")}{" "}
-                  <Link
-                    isUnderline
-                    variant="secondary"
-                    href="https://api.stellar.expert/explorer/testnet/asset-list/top50"
-                    target="_blank"
-                    rel="noreferrer"
-                  >
-                    Stellar Expert's top 50 assets list
-                  </Link>
-                  <br />
-                  <Link
-                    isUnderline
-                    variant="secondary"
-                    href="https://www.freighter.app/faq"
-                  >
-                    {t("Learn more")}
-                  </Link>
+                <div className="TokenWarning__flag__content">
+                  {t("This is a relatively new asset")}
```

Comment:
This "relatively new asset" is hardcoded for now (as all the assets are "relatively new" 😄 ). I'm looking into how we can actually figure out if an asset is now

Reply:
We could index for new token deployments in the future but yeah not sure what else we could do right now.